Drivers/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal.c

```diff
@@ -155,8 +155,8 @@ HAL_StatusTypeDef HAL_Init(void)
 HAL_StatusTypeDef HAL_DeInit(void)
 {
   /* Reset of all peripherals */
-  __HAL_RCC_AHB3_FORCE_RESET();
-  __HAL_RCC_AHB3_RELEASE_RESET();
+  // __HAL_RCC_AHB3_FORCE_RESET();
+  // __HAL_RCC_AHB3_RELEASE_RESET();
```

Comment:
Are you using something in AHB3 during the bootloader stuff that you can't reset? Or is this just hanging around from when you were trying to get the jump to work in the first place?

Reply:
Thought I put a comment there, but I guess not. There wasn't any startup configuration / deinitialization routine I could put together that didn't result in the H750 just restarting immediately upon reaching those two lines of code. I'm not sure what underlying system interaction caused it, but it was very consistent. Since it defeats the purpose of deinitialization (why would you ever do it unless you wanted to avoid a reset?), I though it would be fine to comment it out entirely.

Reply:
Interesting. Yeah, could be something you're using during the bootloader that trips out on AHB3 reset. Since these aren't _our_ files, and we may upgrade, at some point. Can you make a note in the `Drivers/notes.md` file with the change. So that we don't forget to replicate it when we update HAL versions or anything.